src/Dhall/Import.hs

```diff
@@ -411,11 +403,11 @@ canonicalize (File hasHome0 file0:paths0) =
 
         -- `clean` will resolve internal @.@/@..@'s in @currPath@, but we still
         -- need to manually handle @.@/@..@'s at the beginning of the path
-        combine url path = case Filesystem.stripPrefix ".." path of
+        combine url path = case List.stripPrefix ".." path of
```

Comment:
`Data.List.stripPrefix` behaves slightly differently from `Filesystem.stripPrefix`:

```haskell
>>> Filesystem.Path.CurrentOS.stripPrefix ".." "../foo"
Just (FilePath "foo")
>>> Data.List.stripPrefix ".." "../foo"
Just "/foo"
```
I believe this should be `List.stripPrefix "../" instead